custom-login/src/Home.jsx

```diff
@@ -89,7 +89,7 @@ const Home = () => {
           <p>
             Once you have downloaded and started the example resource server, you can visit the
             {' '}
-            <a href="/messages">My Messages</a>
+            <a href="/messages"> My Messages</a>
```

Comment:
Space needed since it'll render the text like `you can visit theMy messages`

Reply:
How about adding `&nbsp;` after "the"

Reply:
And remove `{ ' ' }`

Reply:
Isn't `{ ' ' }` preferable, since it allows the browser to wrap naturally?